### DISCUSSION TITLE
New UI Changes and Section Links

### DISCUSSION THREAD

**Discussion by TravisPeterson21** [GitbookIO/community#518] [2. Published content]
All my section links are rendering as the first page in that section so instead of links that read "User Guide", "Developer Guides", "Release Notes", they all read as the name of the first page which are "Getting Started" and "Introduction". 
![image](https://github.com/GitbookIO/community/assets/54338858/bfd33d9b-2cac-4e84-9541-9ea70aae7eb1)
![image](https://github.com/GitbookIO/community/assets/54338858/ef0bdf1c-b160-45cc-880d-a598eb795724)

Is there a way to link to the sections like the previous version?

**Comment by taszap** [comment]
Hi @TravisPeterson21 - we are tracking this internally. 
We removed the icons but we are looking at improving the design of this (including removing/adjusting the wording "space", "page" etc.)

**Comment by aikoii** [comment]
Hey @TravisPeterson21! The group links are now showing the group title correctly 👍
